gateway/src/main/scala/filodb/gateway/conversion/InputRecord.scala

```diff
@@ -47,7 +47,7 @@ trait InputRecord {
  * the partition key hash.
  * The tags should NOT include the metric name.
  */
-case class PrometheusInputRecord(tags: Map[String, String],
+case class PrometheusInputRecord(tags: collection.mutable.Map[String, String],
```

Comment:
No, let's not make the input mutable, that leads to a whole class of potential bugs where value could change underneath as we are encoding.

Reply:
Please ignore the above change. It was part of an old commit where i was testing something.